tests/Unit/NumericalAlgorithms/LinearSolver/ConjugateGradient/Test_ConjugateGradientAlgorithm.cpp

```diff
@@ -3,6 +3,8 @@
 
 #define CATCH_CONFIG_RUNNER
 
+#include "tests/Unit/TestingFramework.hpp"
```

Comment:
Do you happen to know what of this header is needed? Looking at the header itself I think that this works because of one of the includes in the `TestingFramework.hpp` rather than something define in that header.

Reply:
Following @wthrowe's explanation, this seems to work because Parallel/Abort.hpp includes charm++.h, which some more levels down includes converse.h.

Should I remove it? The [guide lines for unit tests](https://spectre-code.org/writing_unit_tests.html) say
> `tests/Unit/TestingFramework.hpp` must always be the first include in the test file.
but looking at the code this seems to be ignored in most cases.

Reply:
13/364 files matching Test_*.cpp isn't really "most cases".

I guess the real rule would be that it must be included first in any file that uses it.  Most of the files lacking the include are standalone executables.  (We have more of these than we probably should.)  The rest are files testing only with static assertions.

Reply:
Yes, I don't think any of the more integration style tests that have their own executable should need to include that header.